Header header
float32 drone_timeuint16 tag
uint16 size
float32 time_szo
float32 time_corners
float32 time_compute
float32 time_tracking
float32 time_trans
float32 time_update
float32[] time_custom
